lib/web_ui/test/compositing_test.dart

```diff
@@ -173,7 +173,7 @@ void main() {
       expect(picture.buildCount, 1);
       expect(picture.updateCount, 0);
       expect(picture.applyPaintCount, 2);
-    });
+    }, skip: (browserEngine == BrowserEngine.firefox));
```

Comment:
What's the plan for fixing these? Should we file issues?

Reply:
I added an issue already. I even added a project for these.

Reply:
Cool. Can you please add TODO with a link to the issues. Example: https://github.com/flutter/flutter/blob/master/packages/flutter/test/cupertino/context_menu_test.dart#L122

Reply:
Done. (will sync soon)

Note: As far as I see in our code base only minimal number of TODO's have issues next to them. interesting fact.

Reply:
Yes, those are from the days when Jonah was attempting to bootstrap our Web support early on. In those days barely anything worked, so it was excusable to not be thorough, but now we want to get our testing story straight. I cleaned up a lot of these this quarter, but there are many leftovers. To any new ones I've been adding I attached TODOs and links ([example](https://github.com/flutter/flutter/commit/21a18fbe163527a1acc88660abe45c89ef402498), [example](https://github.com/flutter/flutter/commit/903ebd714d2209532012349d814161865bfbb710#diff-970e6be775fd82bcfd4bf2662d840747R262)).